lib/mozart_fetcher/decoder.ex

```diff
@@ -0,0 +1,28 @@
+defmodule MozartFetcher.Decoder do
```

Comment:
this module needs tests!

Reply:
ah you probably told me that you were going to add them, ignore in case

Reply:
Yes yes, will be adding was just to get a quick POC so I could test the branch for performance improvements.